directives/questionnaire.py

```diff
@@ -149,6 +158,8 @@ class QuestionMixin:
     def create_question(self, title_text=None, points=True):
         env = self.state.document.settings.env
         env.question_count += 1
+        if points and len(self.arguments) > 0:
+            env.aplus_quiz_total_points += int(self.arguments[0])
```

Comment:
`env.aplus_quiz_total_points` could be updated later in the same if statement that saves the points in data (near the end of this method). It feels a little unnecessary to repeat the same if condition here.